legacy-sdk/whirlpool/tests/utils/v2/token-2022.ts

```diff
@@ -197,7 +197,7 @@ async function createMintInstructions(
 
     // ConfidentialTransfer
     // [March 6, 2024] getTypeLen(ExtensionType.ConfidentialTransferMint) return 97, but 65 (2 pubkey + 1 bool) is valid
-    // https://github.com/solana-labs/solana-program-library/blob/d72289c79a04411c69a8bf1054f7156b6196f9b3/token/js/src/extensions/extensionType.ts#L74
```

Comment:
This has since been fixed: https://github.com/solana-labs/solana-program-library/blob/55f3d4d67a3be014e0093a7be81bfdaf3b978b00/token/js/src/extensions/extensionType.ts#L81 so we can use the spl-token sdk to get the correct size